package.json

```diff
@@ -9,46 +9,64 @@
   "description": "Daily fork of WebRTC for React Native",
   "license": "MIT",
   "homepage": "https://github.com/daily-co/react-native-webrtc",
-  "types": "index.d.ts",
   "keywords": [
     "react-component",
     "react-native",
     "ios",
     "android",
     "webrtc"
   ],
-  "main": "src/index.js",
+  "react-native": "src/index.ts",
+  "types": "./index.d.ts",
+  "main": "lib/commonjs/index.js",
+  "module": "lib/module/index.js",
   "dependencies": {
+    "adm-zip": "0.5.9",
     "base64-js": "1.5.1",
+    "debug": "4.3.4",
     "event-target-shim": "6.0.2",
     "tar": "6.1.11",
-    "uuid": "3.4.0",
-    "@types/react": "*",
-    "@types/react-native": "*"
+    "@types/react": "17.0.38",
```

Comment:
Wonder if the way they did it upstream might be preferable: https://github.com/react-native-webrtc/react-native-webrtc/compare/106.0.4...daily-co:react-native-webrtc:origin/main-synced-to-106.0.4#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L42

Reply:
[This commit](https://github.com/daily-co/react-native-webrtc/commit/59e6db05b8301399757e7d3a258fef0beb242010) explains why this was added in the first place. Since the TS types are now included directly in the react-native-webrtc project (as opposed to being provided through DefinitelyTyped) I wonder if that changes things...

Reply:
If we do this change, we start to receive errors when trying to build react-native-daily-js:

```
src/DailyMediaView.tsx:3:43 - error TS2307: Cannot find module 'react-native' or its corresponding type declarations.
3 import { ViewStyle, View, Platform } from 'react-native';
```

We receive other errors also related to this one. 

So I would rather make this change in a future PR, if we wish to remove those type dependencies from react-native-webrtc, and include them inside react-native-daily-js, so we don't include more this possible error variable on this one.

Reply:
I agree with your choice.